scripts/stereo_depth_estimation.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright 2020 RT Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rospy
+import rospkg
+import cv2
+import numpy as np
+import copy
+import glob
+import message_filters
+from sensor_msgs.msg import Image
+import rosparam
+from cv_bridge import CvBridge, CvBridgeError
+
+class StereoDepthEstimator:
+    def __init__(self):
+        self._cv_bridge = CvBridge()
+        self._captured_img_l = None
+        self._captured_img_r = None
+        sub_img_l = message_filters.Subscriber("/camera_l/image_rect_color", Image)
+        sub_img_r = message_filters.Subscriber("/camera_r/image_rect_color", Image)
+        self.mf = message_filters.ApproximateTimeSynchronizer([sub_img_l, sub_img_r], 100, 10.0)
+        self.mf.registerCallback(self._img_callback)
+        self._pub_depth_img = rospy.Publisher("/depth/image_rect", Image, queue_size=1)
+        self._captured_img_width = rosparam.get_param("/csi_cam_0/image_width")
+        self._captured_img_height = rosparam.get_param("/csi_cam_0/image_height")
+        self._is_debug = rosparam.get_param("/jnm_depth_estimator/debug")
```

Comment:
```suggestion
        self._captured_img_width = rospy.get_param("/csi_cam_0/image_width")
        self._captured_img_height = rospy.get_param("/csi_cam_0/image_height")
        self._is_debug = rospy.get_param("~debug")
```

`debug`はプライベートパラメータとして宣言しているのでノード名に左右されずに値を取得できるようにしておいたほうがいいと思います。
http://wiki.ros.org/rospy/Overview/Parameter%20Server